validate.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+import argparse
+import os.path
+import subprocess
+import tempfile
+
+MODULES = (
+    '_uuid',
+    'bz2',
+    'ctypes',
+    'curses',
+    'dbm.ndbm',
+    'gzip',
+    'hashlib',
+    'lzma',
+    'readline',
+    'sqlite3',
+    'ssl',
+    'uuid',
+    'venv',
+    'zlib',
+)
+
+
+def test_can_import_modules(py: str) -> None:
+    subprocess.check_call((py, '-c', f'import {",".join(MODULES)}'))
+
+
+def test_can_make_ssl_request(py: str) -> None:
+    prog = '''\
+import urllib.request
+urllib.request.urlopen("https://pypi.org/simple").read()
```

Comment:
That's a lot of data to be fetching, can we do something smaller/faster like... `https://www.google.com/robots.txt`?